packages/conf-g++/conf-g++.0.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "trefis@janestreet.com"
+homepage: "http://gcc.gnu.org"
+authors: "http://www.gnuplot.info/faq/faq.html"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+license: "https://gcc.gnu.org/install/gfdl.html"
+build: [
+  ["which" "g++"]
```

Comment:
I reverted that and just added the depexts to re2.
I'll let someone who knows this stuff do the conf- package properly.